ambari-server/src/test/java/org/apache/ambari/server/security/authorization/AmbariAuthorizationFilterTest.java

```diff
@@ -60,6 +61,11 @@
 import junit.framework.Assert;
 
 public class AmbariAuthorizationFilterTest {
+  @Before
+  public void setUp() {
+    SecurityContextHolder.getContext().setAuthentication(null);
```

Comment:
Technically, this is not the correct solution since the test case appears to use a mocked `SecurityContext`, not the one from the `SecurityContextHolder`. See `org.apache.ambari.server.security.authorization.AmbariAuthorizationFilterTest#performGeneralDoFilterTest`.

I think we need to remove the mocked SecurityContext and use the one provided by Spring.  For example, see `org.apache.ambari.server.controller.internal.UserResourceProviderTest#deleteResourcesTest`.  

In cases like the referenced method, we can do the following rather than create a mocked SecurityContext:
```
    SecurityContextHolder.getContext().setAuthentication(authentication);
```

Since the test case explictly sets the authenticated user before performing the test, there is no real need to explicitly clear it in the _`before`_ method.